lib/PrivilegedDonorContacts/PrivilegedDonorContactsLookup.js

```diff
@@ -12,11 +12,16 @@ import { PRIVILEGED_CONTACT_URL } from './constants';
 export const PrivilegedDonorContactsLookup = ({
   onAddContacts,
   orgId,
-  searchButtonStyle,
+  searchButtonStyle: searchButtonStyleProp,
   searchLabel,
 }) => {
   const stripes = useStripes();
 
+  const canAddContact = stripes.hasPerm('ui-organizations.privileged-contacts.edit');
+  const searchButtonStyle = canAddContact ? searchButtonStyleProp : 'disabled';
+
+  console.log('searchButtonStyle', { searchButtonStyle, canAddContact });
```

Comment:
@alisher-epam 
Do you need this console statement?

Reply:
Also, update changelog if needed.

Reply:
The is for the existing code base.